code_annotations/cli.py

```diff
@@ -88,13 +88,13 @@ def django_find_annotations(
                     click.secho("---------------------------------", fg="red")
                     click.echo("\n".join(searcher.errors))
                     # If there are any errors, do not continue
-                    exit(-1)
+                    sys.exit(-1)
```

Comment:
Pre-existing problem, but `-1` seems like an odd choice of code given that:

> Most systems require it to be in the range 0–127, and produce undefined results otherwise.

https://docs.python.org/3.5/library/sys.html#sys.exit

Reply:
Agreed, I am changing all of these occurrences to use 1 instead.